templates/clientFacade.gotmpl

```diff
@@ -107,6 +113,12 @@ type TransportConfig struct {
     OrgID int64
     // TLSConfig provides an optional configuration for a TLS client
     TLSConfig *tls.Config
+    // NumRetries contains the number of attempted retries
+    NumRetries int
```

Comment:
Do we have a default for NumRetries?

Reply:
I wanted to determine whether users have to set this first and how likely retries are bypassed altogether since it's not configured.

Reply:
This field is optional so the client will not attempt any retries if this field is not set :)